src/main/java/com/airbnb/billow/ARNGenerator.java

```diff
@@ -33,7 +34,10 @@ private static String loadAccountNumber(AmazonIdentityManagementClient iamClient
         String userARN = null;
 
         try {
-            GetUserResult getUserResult = iamClient.getUser();
+            final String USER_NAME = "MikeCurtis";
```

Comment:
I agree, let's make sure the project remains configurable. I agree that putting the account number as one of the config parameters makes sense